doc/api/perf_hooks.md

```diff
@@ -432,70 +432,104 @@ console.log(h.percentile(50));
 console.log(h.percentile(99));
 ```
 
-### Class: Histogram
+## perf_hooks.monitorRequestWrapLatency()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Histogram}
+
+Creates a `Histogram` object that reports the latency of livuv
+request handles. A request handle is an async resource within
+Node.js that tracks the state of an individual asynchronous
+operation (e.g. a file read, a DNS lookup, etc). The latency
+of a request is the length of time from when the request is
+passed off to the event loop to when the corresponding callback
+is invoked.
+
+The histogram effectively measures how long the event loop is
+taking to perform and resolve individual asynhronous operations,
+without differentiating between the types of operations
+measured.
+
+```js
+const [ monitorReqWrapLatency ] = request('perf_hooks');
+const h = monitorReqWrapLatency();
+h.enable();
+// Do something.
+h.disable();
+console.log(h.min);
+console.log(h.max);
+console.log(h.mean);
+console.log(h.stddev);
+console.log(h.percentiles);
+console.log(h.percentile(50));
+console.log(h.percentile(99));
+```
+
+## Class: Histogram
```

Comment:
Since this section is on the main level now, should it be placed before the other classes, ABC-wise?